ambry-router/src/main/java/com.github.ambry.router/PutManager.java

```diff
@@ -277,8 +275,8 @@ public void run() {
             Thread.sleep(sleepTimeWhenIdleMs);
           }
         }
-      } catch (Throwable e) {
-        logger.error("ChunkFillerThread received an unexpected error: ", e);
+      } catch (InterruptedException e) {
```

Comment:
we only need to catch `InterruptedException` now. `Throwable` was an overkill, and `Exception` gets caught within `fillChunks()`.